### PR TITLE
Upgrade to 1.1.2

### PR DESCRIPTION
**Pull request by Thovi98**
## Problem
- New version 1.1.2 is out.

## Solution
- Upgrade to 1.1.2, based on https://github.com/YunoHost-Apps/funkwhale_ynh/pull/143

I’ve not tested yet but must be okay.
I’ve updated `check_process` by removing the upgrade from 0.19.1 because it’s an old release and only available for Yunohost < 4.0 if I remember correctly. Don’t hesitate to say me if I’m wrong to remove it. 

## PR Status
- [x] Code finished.
- [ ] Tested with Package_check.
- [x] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Package_check results
---
* An automatic package_check will be launch at https://ci-apps-dev.yunohost.org/, when you add a specific comment to your Pull Request: "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!"*

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: check_process [hunk 1]
```diff
@@ -16,8 +16,6 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
-		# 0.19.1
-		upgrade=1	from_commit=4d4bf0981e2d3a00eaae6b19bd54344fe55d6530
```
Comment (yalh76) [inline check_process:20]: Why removing test upgrade from 0.19.1 ?

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Review by yalh76 — APPROVED** [review]
LGTM :)

**Comment by Thovi98** [comment]
Thank you, but I can’t, I’ve no rights here 😇

**Pull request merged**